polyform.md

```diff
@@ -26,14 +26,16 @@ The licensor grants you a patent license for the software that covers patent cla
 
 Any noncommercial purpose is a permitted purpose.
 
+## Personal Uses
+
+Personal use for research, experiment, and testing for the benefit of public knowledge, without any anticipated commercial application, personal study, private entertainment, hobby projects, amateur pursuits, or religious observance is for a permitted purpose.
```

Comment:
The wording seems awkward here. I'd have the `...anticipated commercial application...` wording last.

Reply:
Addressed in 1474eeb4c666942458f5dbc5682c1b6048e4787a